node/typings/fullfilmentSimulation.ts

```diff
@@ -100,23 +102,34 @@ interface Address {
 export interface SimulationInput {
   postalCode: string
   geoCoordinates: string[]
-  country: string
-  items: CartItem[]
+  country: string | null
+  items: Item[]
+  priceTables: any[]
+  marketingData: any
+  clientProfileData: ClientProfileDetail | null
+  shippingData: ShippingData | null
+  paymentData: any
+  orderFormId: string | null
+  isCheckedIn: boolean
+  storeId: string | null
+  selectedSla: string
+  checkedInPickupPointId: string | null
 }
 
-interface CartItem {
+interface ShippingData {
+  state: string
+  city: string
+  neighborhood: string
+  street: string
+  isFOB: boolean
+  selectedAddresses: any[]
+  logisticsInfo: LogisticsInfo[]
+}
+
+interface Item {
   id: string
   seller: string
-  sku: string
```

Comment:
Is this removed data coming?

Reply:
As far as I know, it's not. We tested on `famstore`, to make sure of which are the fields that are actually coming.